keg/db/__init__.py

```diff
@@ -8,6 +8,19 @@
 
 class KegSQLAlchemy(fsa.SQLAlchemy):
 
+    def apply_driver_hacks(self, app, info, options):
+        """Override some driver specific settings"""
+        super(KegSQLAlchemy, self).apply_driver_hacks(app, info, options)
+
+        engine_opts = app.config.get('KEG_DB_ENGINE_OPTIONS', {})
+
+        # Turn on SA pessimistic disconnect handling by default:
+        # http://docs.sqlalchemy.org/en/latest/core/pooling.html#disconnect-handling-pessimistic
+        engine_opts.setdefault('pool_pre_ping', True)
+
+        if engine_opts:
```

Comment:
by using setdefault() just above, this `if` is not necessary.

Reply:
Thanks, fixed.